README.md

```diff
@@ -1,59 +1,76 @@
--------
-### ⚠️  PLEASE READ THE [INSTRUCTIONS](/INSTRUCTIONS.md) FOR GUIDELINES ON HOW TO START YOUR PACKAGE.
-> Don't forget to remove this warning while updating this README.
--------
-
-# {package-name}
+# react-split-text
 
 [![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][build-status-image]][build-status-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
 
-[npm-url]:https://npmjs.org/package/@moxy/{package-name}
-[downloads-image]:https://img.shields.io/npm/dm/@moxy/{package-name}.svg
-[npm-image]:https://img.shields.io/npm/v/@moxy/{package-name}.svg
-[build-status-url]:https://github.com/moxystudio/{package-name}/actions
-[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/{package-name}/Node%20CI/master
-[codecov-url]:https://codecov.io/gh/moxystudio/{package-name}
-[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/{package-name}/master.svg
-[david-dm-url]:https://david-dm.org/moxystudio/{package-name}
-[david-dm-image]:https://img.shields.io/david/moxystudio/{package-name}.svg
-[david-dm-dev-url]:https://david-dm.org/moxystudio/{package-name}?type=dev
-[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/{package-name}.svg
+[npm-url]:https://npmjs.org/package/@moxy/react-split-text
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/react-split-text.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/react-split-text.svg
+[build-status-url]:https://github.com/moxystudio/react-split-text/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/react-split-text/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/react-split-text
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-split-text/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-split-text
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-split-text.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-split-text?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-split-text.svg
 
-{package-description}
+A react component that splits a sentence into words.
```

Comment:
This might be a little confusing because a sentence is a set of words 😂
I would say something like:

> A react component that wraps each word of a sentence into a `<span>` element

or if you want to use `split` since the package name has `split` on it

> A react component that splits a sentence by wrapping each word into a `<span>` element

What do you think?
If you agree with this suggestion, don't forget to change `package.json` description as well.

Reply:
> This might be a little confusing because a sentence is a set of words 😂

I know but nothing better was coming to my mind so I went with that 😂

I'll go with the latter, thanks 👍

Reply:
Actually I'll go with the first one as it seems more on point.